src/components/sbb-tab-group/sbb-tab-group.e2e.ts

```diff
@@ -83,5 +83,45 @@ describe('sbb-tab-group', () => {
 
       expect(tab).toHaveAttribute('active');
     });
+
+    it('wraps around on arrow left arrow key navigation', async () => {
+      await page.keyboard.down('Tab');
+      await page.keyboard.down('ArrowLeft');
+      await page.keyboard.down('ArrowLeft');
+      const tab = await page.find('sbb-tab-group > sbb-tab-title#sbb-tab-4');
+
+      expect(tab).toHaveAttribute('active');
+    });
+  });
+
+  it('activates the first enabled tab if exceeds the length of the tab group', async () => {
+    page = await newE2EPage();
+    await page.setContent(`<sbb-tab-group initial-selected-index="2">
+              <sbb-tab-title id="sbb-tab-1">Test tab label 1</sbb-tab-title>
+              <div>Test tab content 1</div>
+              <sbb-tab-title id="sbb-tab-2">Test tab label 2</sbb-tab-title>
+              <div>Test tab content 2</div>
+            </sbb-tab-group>`);
```

Comment:
Could be better formatted, like
```suggestion
    await page.setContent(`
        <sbb-tab-group initial-selected-index="2">
          <sbb-tab-title id="sbb-tab-1">Test tab label 1</sbb-tab-title>
          <div>Test tab content 1</div>
          <sbb-tab-title id="sbb-tab-2">Test tab label 2</sbb-tab-title>
          <div>Test tab content 2</div>
        </sbb-tab-group>
    `);
```
Same for next test.